lib/posix-sysinfo/include/sys/sysinfo.h

```diff
@@ -33,6 +33,10 @@ extern "C" {
 
 #define SI_LOAD_SHIFT 16
 
+/**
+ * Structure used by the `sysinfo()` system call.
+ * Follows Linux conventions.
```

Comment:
I don't know if we should have this line for every structure / call.

I think it would be better to have the opposite when needed.

Reply:
I would leave that in. Compatibility with the Linux kernel is not expected by default, so it's fine to let people know when it's the case.